wip

0x5d opened the pull request:
## Checklist
- [ ] Reference related [issue](https://github.com/vectorizedio/redpanda/issues)
- [ ] Update [PendingReleaseNotes.md](https://github.com/dotnwat/redpanda/blob/dev/PendingReleaseNotes.md), if relevant

When referencing a related issue, remember to migrate duplicate stories from the
external tracker. This is not relevant for most users.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/vectorizedio/redpanda?pullRequest=81) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/vectorizedio/redpanda?pullRequest=81) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/vectorizedio/redpanda?pullRequest=81) it.</sub>

0xdiba commented on .github/workflows/rpk-build.yml:
```diff
@@ -8,30 +8,124 @@
+      working-directory: src/go/rpk/
       run: go test -v ./...
+
+    - name: Build Linux
```
Should the two `Build XXX` steps be described in a matrix `[linux, darwin]`?

0x5d replied:
Oh cool! I'll look into it.

0xdiba commented on .github/workflows/rpk-build.yml:
```diff
@@ -8,30 +8,124 @@
 
-    - name: Publish
-      run: echo "TODO"
+    - name: Upload build artifacts
```
Optional but this step can also be guarded with : `if: startsWith(github.ref, 'refs/tags/release-')`

0xdiba commented on .github/workflows/rpk-build.yml:
```diff
@@ -8,30 +8,124 @@
+        draft: true
+        prerelease: false
+
+    - name: Upload Linux OS rpk to release
```
Upload steps can utilize the same matrix strategy as the builds

0xdiba commented on .github/workflows/rpk-build.yml:
```diff
@@ -8,30 +8,124 @@
+  push:
+    tags: 'release-*'
+    branches:
+      - '*'
```
main/dev or not?

0x5d commented:
@0xdiba I screwed up and closed this by accident, so I had to re-create the PR because I had force-pushed to the source branch. Please check #102 , I believe I addressed all your feedback.
